ui/components/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { render } from "@testing-library/react";
+import { withTheme } from "../../lib/test-utils";
+import Link from "../Link";
+
+describe("Link", () => {
+  it("doesn't create a link for oci links", () => {
+    const { container: link } = render(
```

Comment:
I have no idea how this works but I'm not mad at it. Might be worth adding isAllowed to the isHTTP tests in utils, or deleting the isHTTP tests altogether if we don't need to use it anymore